Bottom: 4f3d293931fa91bad71e64e1774e528f75e4969c
Top:    b57b4d9bba3bcff69b6642261aa1f69e9b278c0b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-21 14:13:31 +0000

Disable polygon heritage


---

diff --git a/src/polygon1.c b/src/polygon1.c
index a9ac914..d6f8ead 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2415,8 +2415,6 @@ add_dummy_descriptors_at_point (Vector point, char poly, CVCList * list, POLYARE
   POLYAREA *b = bfst;
   PLINE *cur;
 
-  pcb_fprintf (stderr, "LOOKING FOR VERTICES AT POINT (%$mn, %$mn)\n", point[0], point[1]);
-
   assert (b != NULL);
   do
     {
@@ -2827,8 +2825,10 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
   *res = NULL;
 
   /* Make copies for tracking polygon parentage (DEBUG) */
-  if (!poly_M_Copy0 (&a_copy, a) || !poly_M_Copy0 (&b_copy, b))
-      return err_no_memory;
+//  if (!poly_M_Copy0 (&a_copy, a) || !poly_M_Copy0 (&b_copy, b))
+//      return err_no_memory;
+  a_copy = NULL;
+  b_copy = NULL;
 
   /* Move the parentage information over onto the copy */
   if (a_copy != NULL)
